instrumentation/http-tests/src/main/java/brave/http/ITHttpServer.java

```diff
@@ -265,16 +227,4 @@ private void addsErrorTagOnException(String path) {
         .extracting(b -> b.key)
         .contains(Constants.ERROR);
   }
-
-  @Test
-  public void httpPathTagExcludesQueryParams() throws Exception {
-    String uri = "/foo?z=2&yAA=1";
-
-    Request request = new Request.Builder().url(url(uri)).build();
-    try (Response response = client.newCall(request).execute()) {
-      assertThat(response.isSuccessful()).isTrue();
-    }
-
-    assertReportedTagsInclude(TraceKeys.HTTP_PATH, "/foo");
-  }
 }
```

Comment:
refactored this file to pay for some of the lines of code, although it didn't really pay for many of them :P